### DISCUSSION TITLE
Table 16, Panel 1

### DISCUSSION THREAD

**Discussion by pkgw** [cssi-pi-community/2022-meeting#30] [Panel 1 roundtable (Monday morning)]
This is the thread for table 16!

**Comment by guoningchen** [comment]
Having a centralized plan like the Broadening Participation in Computing initiated by CISE for most Computer Science Departments in the state may help .

**Reply by pkgw** [reply]
Link: https://www.nsf.gov/cise/bpc/

**Comment by guoningchen** [comment]
We are probably focusing too much about the recruitment and retention of students to stay at STEM fields and forget to attract people who already graduated with degrees that are not in STEM fields.

**Comment by mgzentner** [comment]
Consider conducting a study on what a diverse employee pool values in a position, recognizing that academia will not likely compete on salary with industry.

**Comment by jkmorin** [comment]
It is important that we consider DEI as an empowerment not as outreach. We can learn much from different cultural communities.We must stop judging totally by publish or perish. We must educatie a future workforce that understands the importance of the arts and humanities with STEM, as well as the richness that is demonstrated by a multicultural world.
